### ISSUE TITLE
Formulate "Strong" Attacks [Dom]

### ISSUE THREAD

**Issue by domenicrosati** [domenicrosati/training-time-domain-authorization#5]
We need at least the following fine-tuning attack settings added:
- Sample Size
- Learning Rate
- Batch Size
- Optimizer (Plus optimizer hyperparams)

But we should also consider stronger or alternative attacks for example:
- LoRA and other PEFT methods
- Task Arithematic and Model Merging
- Others?

The outcome of this is both:
- Fill out a super strong attack plan on https://docs.google.com/document/d/1YW26CdAKv06uc2CN09vf-w9RpkQbYU5yBMkaQBWkH84/edit
- Add these settings to the code
